### ISSUE TITLE
[regression] Variable data no longer respected in 

### ISSUE THREAD

**Issue by brittyazel** [indico/indico#5766]
**Describe the bug**
In the notification emails sent to registrants, variable data elements (e.g, {first_name}) used to be respected, but with the conversion to using markdown it appears that they no longer are. We just had a somewhat embarrassing moment when we received a bunch of reports that our event registration emails went out as the literal "Dear {first_name}" instead of say, "Dear Jane".

I believe this _may_ have been introduced in PR #4741, but I can't be sure without going back and testing it on a test instance.

That being said, it would be great if we could have this functionality back, and/or if there's a new way to do the variable data elements in markdown I would love to learn. Thanks.

**Additional context**
We're running Indico v3.2.3

**Comment by ThiefMaster** [comment]
Are you *absolutely* sure we ever supported placeholders in those emails? IIRC only custom emails sent to specific registrants allow for placeholders...

**Comment by brittyazel** [comment]
Hrm... I thought so...

Our team has all of the text we use for these emails saved, and in or records we have the text we use for the registration notification emails going back a few years all using the variable data. That said, it could very well be that we were making an assumption this whole time and 'all' of our notification emails have had this bug and we just never noticed and we might have never thought to check...

**Comment by brittyazel** [comment]
Nope, I looked at some old logs and you're right, it was always not work. Now I'm really embarrassed, yikes.

Question, why not allow for variable data for things like notification emails?

**Comment by ThiefMaster** [comment]
It's just a snippet of text that goes in another wise predefined email. I think if we were to allow placeholders there, this should be something more thorough to allow more customization than just that one paragraph.

**Comment by brittyazel** [comment]
Interesting, well alright. I agree, it would be great to have something more than the text snippet. Count me in as supporting that feature! One of the best things about Indico (that sets it apart from other event platforms) is the control we have over the emails that get sent out to participants. Anything that can improve that further is 10/10 in my book.

**Comment by ThiefMaster** [comment]
Would you mind editing this issue to make it a suggestion unless we already have one (not sure).

Alternatively just close it since it's not actually a bug :)
